builtin/providers/openstack/resource_openstack_compute_secgroup_v2.go

```diff
@@ -122,7 +122,7 @@ func resourceComputeSecGroupV2Read(d *schema.ResourceData, meta interface{}) err
 	d.Set("name", sg.Name)
 	d.Set("description", sg.Description)
 	log.Printf("[DEBUG] rulesToMap(sg.Rules): %+v", rulesToMap(sg.Rules))
-	d.Set("rules", rulesToMap(sg.Rules))
+	d.Set("rule", rulesToMap(sg.Rules))
```

Comment:
Yeah, it wasn't exporting even before I added `ruleID`. I added it because I didn't know if it was the name `id` that it didn't like.